test/functional/fixtures/runner/upload/regression-test.js

```diff
@@ -0,0 +1,32 @@
+var expect = require('chai').expect;
+
+
+describe('Upload regression', function () {

```

Comment:
Why this can't be in the upload-test file in the sub-fixture `Regression`, like we have for the server tests?